chain/pool/txpool.go

```diff
@@ -87,6 +89,34 @@ func (tp *TxnPool) AppendTxnPool(txn *transaction.Transaction) error {
 	return nil
 }
 
+func (tp *TxnPool) totalClaim(sender []byte) (common.Fixed64, error) {
+	amountToClaim := common.Fixed64(0)
+	var err error
+	tp.NanoPayTxs.Range(func(_, v interface{}) bool {
+		tx := v.(*transaction.Transaction)
+		var pld transaction.IPayload
+		pld, err = transaction.Unpack(tx.UnsignedTx.Payload)
+		if err != nil {
+			return false
+		}
+		payloadNp := pld.(*pb.NanoPay)
```

Comment:
I think It's better to use `payloadNp, ok := pld.(*pb.NanoPay)` here to avoid panic caused by accidentally put non-nanopay txn into here

Reply:
no payload reading in pool anymore